File: Android/app/src/main/java/com/adsamcik/studentsassistant/Class.java
package com.adsamcik.studentsassistant;

/**
 * Created by adsamcik on 19.09.2015.
 */
public class Class {
    public String name, shortName;

    public Class(String name, String shortName) {
        this.name = name;
        this.shortName = shortName;
    }

    @Override
    public String toString() {
        return name + " (" + shortName + ")";
    }
}

File: Universal/SharedLib/Class.cs
using System;

namespace SharedLib {
    public class Class {
        public string uid;
        public string name;
        public string shortName;

        public Class(string name, string shortName, string uid = "") {
            this.name = name;
            this.shortName = shortName;

            if (string.IsNullOrWhiteSpace(uid))
                this.uid = Guid.NewGuid().ToString();
            else
                this.uid = uid;
        }

        public override string ToString() {
            return name + " (" + shortName + ")";
        }
    }
}
